src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
+
+  if (days >= 7) {
+    sale = 50;
+  } else if (days >= 3) {
+    sale = 20;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. Also, you don't need to add `else` after `if` with `return` as the code after it won't be executed anyway. Refactor the code to return immediately after applying the discount.